### PR TITLE
Book list

### PR DESCRIPTION
**Pull request by jersonlopez**
## Summary
- book list added
- list items linking to book detail

## Screenshot
image

## Trello Card
https://trello.com/c/VW5sr91l/30-listado-de-libros

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -8,29 +8,66 @@
...
-        <div class="book-description">
-          <h3 class="subtitle bold">Año de publicación:</h3>
-          <span class="default-text">(Año de publicación)</span>
-        </div>
-      </div>
+  <body class="body">
+    <div class="books">
+      <ul class="book-list">
+        <li>
+          <a href="./book-detail.html" class="facebook">
```
Comment (CrisVega08) [inline maquetado/index.html]: class facebook 👀 ??

File: maquetado/scss/detail.scss [hunk 2]
```diff
@@ -73,3 +75,46 @@
...
+}
+
+.list-title {
+  color: $black;
+  font-size: 16px;
+  line-height: 19px;
+  margin-bottom: 5px;
+}
+
+.facebook {
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss]: I suggest change this name  for something like `cover-page`

File: maquetado/scss/detail.scss [hunk 3]
```diff
@@ -73,3 +75,46 @@
...
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $opaque-black;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  margin-bottom: 35px;
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss]: use shorthand margin: 0 35px 35px 0;

File: maquetado/scss/detail.scss [hunk 4]
```diff
@@ -73,3 +75,46 @@
   font-size: 24px;
   line-height: 28px;
 }
+
+.book-list {
```
Comment (CrisVega08) [inline maquetado/scss/detail.scss]: These styles make no sense in this file, create new style file for index page

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: maquetado/index.html (hunk 1 above)
Comment (jersonlopez) [reply maquetado/index.html]: because facebook = "portada"

File: maquetado/scss/detail.scss (hunk 3 above)
Comment (jersonlopez) [reply maquetado/scss/detail.scss]: why? if i just using margin in one side

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**